tutorials/migration.md

```diff
@@ -463,78 +391,71 @@ cp <openHAB 1.x conf>/configurations/sitemaps/* <openHAB 2 conf>/sitemaps/*
 
   * Static webview files are now located in `<openhab 2 conf>/html` instead of  `/usr/share/openhab/webapps`
```

Comment:
According to #189 we should use a consistent `$FOLDER` notation for all articles.